docs/DeveloperGuide.md

```diff
@@ -1164,8 +1204,29 @@ testers are expected to do more *exploratory* testing.
 
 ***
 
-### UI Testing
-
-#### Buttons
+### Graphic User Interface Testing
+
+#### Menus
+   1. Test the 5 menu items on the bar
```

Comment:
Might want to specify the state of AD, from the pre-req seems to be default so it should be like "Newly opened AD, all students displayed in student list and result display is blank.